README.md

```diff
@@ -49,18 +49,19 @@ Show usage of `useState` react hook to manage a controlled input
 ### Button Grid
 
 Render a 3 x 3 grid of 9 buttons.
-When component renders, it should select one button at random and make it red, the rest of the buttons should be green. When red button is clicked it should cause another button to turn red at random. When an green button is clicked, it should become red.
+When component renders, it should select one button at random and make it red, the rest of the buttons should be green. When a red button is clicked it should cause another button to turn red at random. When a green button is clicked, it should become red.
+While you keep clicking, all the 9 buttons will be red in the end. 
```

Comment:
I misunderstood it in another way

- 5 is selected, Click 5 -> 5 is deselected and another random number is selected
- 5 is selected , Click 6 -> 5 is deselected and 6 is selected

It was my bad. Just want to add an extra sentence that prevents others from going to the wrong direction.

Reply:
Sorry, I haven't push an update yet. The idea is to make it so that there's only 1 red button at any given time.
Also I'd like to change this exercise to be a little more complicated and try to use `useEffect`.

So, in short, ending up with all red squares is not the goal because that's just too simple.